src/index.html

```diff
@@ -6,9 +6,34 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a class="header__link" href="#home">
+          <img class="logo__img" src="images/logo.png"
```

Comment:
<img width="1343" alt="Screenshot 2023-08-17 at 09 36 19" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/9147662a-effb-48c5-8cf3-516c622b1697">

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3722/files#r1296739195
didn't fix it